wordle-web/pages/index.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-card
+    color="secondary text-primary"
+    class="mx-auto mt-8 w-75 pa-8 rounded text-center"
+    elevation="4"
+  >
+    <v-card-title class="text-wrap">
+      Hello There! Welcome to Johanne and Alexa's Pink Wordle Website
+    </v-card-title>
+    <v-card-subtitle class="text-wrap">
+      We made our own "Wordle" game variation called "Pink Wordle" and invite
+      you to play! Have fun! :)
+    </v-card-subtitle>
+    <v-card-actions class="justify-center mx-2 my-4">
+      <v-btn color="secondary" class="bg-primary" to="/game"
+        >Play Pink Wordle!</v-btn
+      >
```

Comment:
```suggestion
      <v-btn color="secondary" class="bg-primary" to="/game">
        Play Pink Wordle!
      </v-btn>
```
The formatter will bite you if you don't put a space between your closing and opening angle brackets.

Reply:
good to know! Thank you :)